I.write-tests/03-remove-vowels-from-array/remove-vowels-in-array.test.js

```diff
@@ -1,11 +1,15 @@
-test('remove vowels from all words in array', function () {
-    // Arrange
+test("remove vowels from all words in array", function() {
+  // Arrange
+  var removeVowelsForWords = require("./remove-vowels-in-array");
+  var words = ["Irina", "Etza", "Daniel"];
 
-    // Act
+  // Act
+  var result = removeVowelsForWords(words);
 
-    // Assert
-})
+  // Assert
+  expect(result).toEqual(["rn", "tz", "dnl"]);
```

Comment:
Did this pass? I'm surprised it would have removed the uppercase "I" in "Irina" as expected?